### PR TITLE
fix: support tailing commas in SQL

### PR DESCRIPTION
**Pull request by Taylor-lagrange**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Because some bug in sqlparser,  sql like `SELECT foo, bar, FROM baz` with tailing commas will be wrongly parse to `SELECT foo, bar, FROM AS baz`

Enable `trailing_commas` in sqlparser to avoid such error.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by waynexia — APPROVED** [review]
LGTM

File: tests-integration/src/tests/promql_test.rs [hunk 1]
```diff
@@ -193,7 +217,7 @@ const AGGREGATORS_CREATE_TABLE: &str = r#"create table http_requests (
 // http_requests{job="app-server", instance="1", group="production"} 0+60x10
 // http_requests{job="app-server", instance="0", group="canary"}   0+70x10
 // http_requests{job="app-server", instance="1", group="canary"}   0+80x10
-const AGGREGATORS_INSERT_DATA: &str = r#"insert into http_requests(job, instance, group, value, ts) values
+const AGGREGATORS_INSERT_DATA: &str = r#"insert into http_requests(job, instance, "group", value, ts) values
```
Comment (WenyXu) [inline tests-integration/src/tests/promql_test.rs:220]: Non-block: why add a quote

**Review by WenyXu — APPROVED** [review]
LGTM,

File: tests-integration/src/tests/promql_test.rs (hunk 1 above)
Comment (waynexia) [reply tests-integration/src/tests/promql_test.rs:220]: `group` is a keyword

**Pull request merged**